src/version.ts

```diff
@@ -1 +1 @@
-export const PACKAGE_VERSION = "0.18.0";
+export const PACKAGE_VERSION = "dev";
```

Comment:
this will get copied from package.json by scripts, and then used for the file-path for saving stuff, so maybe a strategy that yields eg 0.19-dev in situations like this would be good for sanity long term